examples/single_task/write/memory/a10_write_memory.yml

```diff
@@ -8,6 +8,7 @@
       a10_username: "{{ a10_username }}"
       a10_password: "{{ a10_password }}"
       a10_port: "{{ a10_port }}"
-      a10_protocol: "{{ a10_protocol }}}"
-      partition: "shared"
-      destination: "primary"
+      a10_protocol: "{{ a10_protocol }}"
+      a10_partition: "shared"
```

Comment:
The value of `a10_partition` key has changed. It's now (for shared):
```
a10_partition:
  name: shared
  shared: 1
```

or for a user defined partition:
```
a10_partition:
  name: mypartition
  shared: 0
```

I'll be updating the ReadMe soon to indicate these change.

Reply:
That being said. `a10_partition` should not be specified when using `write-memory` as `write-memory` allows you to specify `partition`.